hadoop-crypto/src/test/java/com/palantir/crypto2/hadoop/StandaloneEncryptedFileSystemTest.java

```diff
@@ -102,6 +103,49 @@ public void testReadWrite() throws IOException {
         assertThat(rawFs.exists(new Path(path + FileKeyStorageStrategy.EXTENSION))).isTrue();
     }
 
+    @Test
+    public void testDelete() throws IOException {
+        File rootFolder = folder.newFolder();
+
+        Path path1 = writeData(rootFolder);
+        Path path2 = writeData(rootFolder);
```

Comment:
this seems a bit bogus: you're verifying that `delete` doesn't touch a totally unrelated path?

Reply:
removed in the follow up PR